OZON.ru.js

```diff
@@ -1,604 +1,103 @@
 {
 	"translatorID": "73be930f-5773-41b2-a7a1-37c0eeade92f",
 	"label": "OZON.ru",
-	"creator": "Victor Rybynok",
-	"target": "^https?://www\\.ozon\\.ru",
-	"minVersion": "2.1.9",
+	"creator": "Mikhail Akimov",
+	"target": "http://www.ozon.ru/context/detail/id/\\d{3,}",
```

Comment:
"^https?://www\\.ozon\\.ru/context/detail/id/\\d{3}" This is used for test (not match), so there's no point in matching more than needed, like `{3,}`